cmd/geteduroam/main.go

```diff
@@ -271,7 +272,18 @@ func oauth(p *instance.Profile) {
 	}
 }
 
-func main() {
+func doLocal(filename string) {
+	    b, err := os.ReadFile(filename)
+	    if err != nil {
+		    log.Fatalf("Failed to read local file: %v", err)
+	    }
+	    err = file(b)
```

Comment:
Het zal vast werken, maar wat is of doet file()?

Reply:
https://github.com/geteduroam/linux-app/blob/b6241cbdcf7e13275a1f8598689a68ab224d56ba/cmd/geteduroam/main.go#L223

Reply:
Ah! We haken gewoon later in op wat er al was, maar dan via discovery. Helder.